docs/REST API/rest-api.md

```diff
@@ -6446,7 +6456,10 @@ Creates a script execution request and waits for a result to return (up to a 1 m
 | Name            | Type    | In   | Description                                      |
 | ----            | ------- | ---- | --------------------------------------------     |
 | host_id         | integer | body | **Required**. The host id to run the script on.  |
-| script_contents | string  | body | **Required**. The contents of the script to run. |
+| script_id       | integer | body | The script to run. The current contents of this saved script will be executed on the host. Only one of script_id and script_contents must be provided.  |
+| script_contents | string  | body | The contents of the script to run. Only one of script_id and script_contents must be provided. |
```

Comment:
Just duplicated the changes to the run asynchronously parameters for extra clarity (since folks won't necessarily look at these in order):
```suggestion
| script_id       | integer | body | The ID of the existing saved script to run. Only one of either `script_id` or `script_contents` can be included in the request; omit this parameter if using `script_contents`.  |
| script_contents | string  | body | The contents of the script to run. Only one of either `script_id` or `script_contents` can be included in the request; omit this parameter if using `script_id`. |
```